mingw-w64-hid-bootloader-cli/PKGBUILD

```
@@ -0,0 +1,40 @@
+# Maintainer: fauxpark <fauxpark@gmail.com>
+
+_realname=hid_bootloader_cli
+pkgbase=mingw-w64-${_realname//_/-}
+pkgname=${MINGW_PACKAGE_PREFIX}-${_realname//_/-}
+pkgver=210130
+pkgrel=1
+pkgdesc='LUFA HID Bootloader CLI (mingw-w64)'
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+license=('GPL3')
```

```suggestion
license=('custom:MIT')
```

See https://fedoraproject.org/wiki/Licensing:MIT#Old_Style_with_legal_disclaimer_2 https://github.com/abcminiuser/lufa/blob/master/LUFA/License.txt

---

The HID Bootloader CLI itself seems to be GPL3, not MIT: https://github.com/abcminiuser/lufa/tree/master/Bootloaders/HID/HostLoaderApp
Which makes sense as teensy_loader_cli is GPL3: https://github.com/paulstoffregen/teensy_loader_cli

---

I see, thanks